### PR TITLE
[DRAFT] [Community Tutorial] Intro to Geospatial Analysis

### PR DESCRIPTION
**Pull request by TC25**

### CODE REVIEW

**Comment by TC25** [comment]
I made a draft pull request. This is to mainly test things for the build-a-thon session.

**Comment by gino-m** [comment]
> I made a draft pull request. This is to mainly test things for the build-a-thon session.

Great, thank you!

File: tutorials/Intro-to-geospatial-analysis/index [hunk 1]
```diff
@@ -0,0 +1,426 @@
+---
+title: An Introduction to Geospatial Analysis Using Google Earth Engine
+description: This workshop provides a fundamental overview of remote sensing and geospatial analysis using the JavaScript API of Earth Engine.
+author: TC25
+tags: introductory, GIS, remote sensing, Google Earth Engine
+date_published: 2019-09-14
+---
```
Comment (gino-m) [inline tutorials/Intro-to-geospatial-analysis/index]: Hi TC! What do all the "+" prefixes mean here and throughout? Were you meaning to put the whole content in a table?

Comment (TC25) [reply tutorials/Intro-to-geospatial-analysis/index]: There is no + in the actual markdown. I think it just indicates whether something was added to the file.

**Pull request closed without merging**